tests/unit/test_lazy_imports.py

```
@@ -0,0 +1,159 @@
+    """Run *code* in a fresh Python subprocess and return the decoded JSON result.
+
+    The subprocess has ``src`` on ``PYTHONPATH`` and runs with minimal env so
+    plugin registration does not depend on host device state.
+    """
+    env = os.environ.copy()
+    env["PYTHONPATH"] = str(SRC_DIR)
+    env["INKYPI_ENV"] = "dev"
+    env["INKYPI_NO_REFRESH"] = "1"
+    # Keep test output small — suppress noisy warnings from the inky library.
+    env["PYTHONWARNINGS"] = "ignore"
+    proc = subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)